schema/verifiable-credential/verifiable-credential-schema.json

```diff
@@ -172,7 +212,12 @@
       ]
     },
     "validFrom": {
-      "type": "string"
+      "type": "string",
+      "pattern": ".*(Z|(\\+|-)[0-9][0-9]:[0-9][0-9])"
+    },
+    "validUntil": {
+      "type": "string",
+      "pattern": ".*(Z|(\\+|-)[0-9][0-9]:[0-9][0-9])"
```

Comment:
Are we sure about this regex? Valid from/until are supposed to be dates, and these only seem to restrict the time part. Why not using a regex for the full xsd datatype?

Reply:
IIUC, these regexes are incorrect. The correct regex can be found here:

https://www.w3.org/TR/vc-data-model-2.0/#example-regular-expression-to-detect-a-valid-xml-schema-1-1-part-2-datetimestamp